src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -183,6 +183,10 @@ const useCloudEnvironmentPolling = googleProject => {
         Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: tools.galaxy.appType }),
```

Comment:
@kyuksel Thanks for picking up this review! What I would like to do here is remove the explicit mentioning of Galaxy and Cromwell by iterating through the tools and adding a Disks.list call for each tool that is an app. Then I need to inject the app type into the returned response for each.

Is there any downside to doing each of those calls as its own await so I can easily do it in a loop? Or can you think of some way to do it with the existing Promise.all?

(I also expect to be touching this code again for the Enviornment.js disk fix so that I can share the code between the 2 locations, so I'm also OK with just leaving the explicit mentions in this iteration.)

Reply:
I _think_ I see what you're proposing. I feel like we may want to, instead, invest the time to add the necessary info to Leo's `listDisks()` so we don't need to make multiple `Ajax(signal).Disks.list` calls at the first place.

Reply:
I agree, I think we will be able to do it in a single .list call once the formatted by information is present, so it's not worth trying to do my proposed intermediary improvement.